packages/bot-skeleton/src/services/tradeEngine/trade/Purchase.js

```diff
@@ -28,7 +28,8 @@ export default Engine =>
                 this.store.dispatch(purchaseSuccessful());
                 this.renewProposalsOnPurchase();
                 delayIndex = 0;
-                log(`${localize('Bought')}: ${buy.longcode} (${localize('ID')}: ${buy.transaction_id})`);
+                log(log_types.PURCHASE, { longcode: buy.longcode, transaction_id: buy.transaction_id });
+                // log(`${localize('Bought')}: ${buy.longcode} (${localize('ID')}: ${buy.transaction_id})`);
```

Comment:
Is this comment necessary to remain in the code?

Reply:
Can be delete. Thanks 👍